docs/server-side/v2/api-reference/external-streams/Get-details-for-a-job-id.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Get details for a given RTMP Job ID
```

Comment:
We can improve the copy to avoid internal terms like job and terms like RTMP. Can do that in a follow up PR